package.json

```diff
@@ -50,10 +51,14 @@
     "vite": "^3.1.4",
     "vitest": "^0.2.5"
   },
-  "keywords": [
-    "vite",
-    "vite-plugin",
-    "react-docgen-typescript"
-  ],
+  "peerDependencies": {
+    "typescript": ">= 4.3.x",
+    "vite": "^3.0.0"
```

Comment:
We should open up the vite dependency to include ^4 as well.  Can do it in a separate PR if we want to keep this PR scoped, but it should be done.

Reply:
We should do it in another PR